### DISCUSSION TITLE
Rename Material to GraphicsMaterial

### DISCUSSION THREAD

**Discussion by markushonok** [godotengine/godot-proposals#5215] [Engine Core]
Although the name Material is quite well-established, but looking at it from the other side it looks a bit strange when you see PhysicsMaterial, which is in no way related to Material, but they both have Material in the name. It seems to me that adding "Graphics" to the type name would make it more correct. I noticed this when I needed to keep both types in the same place.
```csharp
public class MonoMaterial : Resource
{
	[Export]
	public Material? GraphicsMaterial { get; set; }

	[Export]
	public PhysicsMaterial? PhysicsMaterial { get; set; }
}
```

**Comment by Mickeon** [comment]
I'd rather rename **PhysicsMaterial** than the core **Material** class that every other _Material_ type inherits from.
